internal/configuration/config.go

```diff
@@ -44,6 +46,16 @@ type Config struct {
 	Daemon bool `yaml:"-"`
 }
 
+type EventsConfig struct {
+	Enabled bool         `yaml:"enabled"`
```

Comment:
For consistency, we should have a top level `enableEvents` flag similar to the `enableDiscovery` flag. And if enabled the `EventsConfig` will have config relevant to collecting events.